src/utils/apiClient.tsx

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+const baseUrl = `${process.env.REACT_APP_API_URL ?? ''}api/`;
```

Comment:
@sniedzielski I would consider `api/` to be a part of the defined `REACT_APP_API_URL` (see also my next comment)

Reply:
ok. I will change this.